Iniciar Sesion y Registrarse

Danieljm11 opened the pull request:
Se crearon las secciones de iniciar sesion y de registrarse para cada una de los usuarios, se creo la carpeta HTML dentro de Public y se movieron los archivos dentro de la carpeta.

Danieljm11 committed:
Iniciar Sesion y Registrarse

Se crearon las secciones de iniciar sesion y de registrarse para cada una de los usuarios, se creo la carpeta HTML dentro de Public y se movieron los archivos dentro de la carpeta.

marvmena committed:
Edición

Se borró el codigo de la pagina de proovedores, ya que no se va hacer, y se agregaron dos html y dos css para comenzar a trabajar dos de mis wireframes.

Danieljm11 committed:
Paginas de Inicio

Se agregaron las paginas de inicio de administrador, vendedor y comprador con sus respectivas imagenes y archivos de CSS

Jhnst2705 committed:
Restablecer contraseña admin, comprador, vendedor. html-css-js
Actualizar perfil admin, comprador, vendedor. html-css-js*
registro productos. html-css

marvmena committed:
Adelanto de las vistas: editarCategoriasAdministrador y inventarioVendedor

Se generaron templates iniciales para las vistas de editarCategoriasAdministrador y inventarioVendedor en donde todavía se deben hacer unos ajustes en cuanto a la validación de los distintos estándares definidos para los estilos.

crisSotoMo reviewed (commented):
Hola, se ve super bien. Tengo algunas preguntas.
1. Esto no es lo que hace el reset?

* {
    margin: 0;
    padding: 0;
}

2. Puede que sea yo que no lo entienda, pero por qué estás haciendo el import aquí y no en el HTML? 
@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
@import url('https://fonts.googleapis.com/css2?family=Archivo+Black&display=swap');

Me avisas para aprobar

marvmena committed:
Cambios finales Leo

Se envian el codigo completo

cherloydfans13141 committed:
Adding my files Daniel M

All the pages I was assigned

cherloydfans13141 committed:
Merge branch 'Desarrollo' of https://github.com/crisSotoMo/PlataformaEcommerce into Desarrollo

crisSotoMo committed:
Organizar estructura

artch92 committed:
Unificación de Proyecto

Se unieron los distintos modulos de cada uno de los integrantes y se genero una versión estandar para que se pueda trabajar en los cambios finales.

crisSotoMo committed:
footer and reportesVendedor fix

cherloydfans13141 committed:
Uploading my updated pages

I have updated all my pages to fit the format, also I'm uploading the js files for the validations needed for the forms.

marvmena committed:
Correciones finales

Se agregaron las correciones del footer, header y se hicieron correciones vistas.

marvmena committed:
Actualizando JS de vistas de restablecerContraseña y editarPerfil

artch92 committed:
Ultimos cambios restablecerContraseña y editarPerfil de cada uno de los usuarios

marvmena committed:
Cambios finales a restablecer contraseña y editar perfil.

cherloydfans13141 committed:
Container format

Updated the format of the containers for the payment method pages

crisSotoMo committed:
Cambios varios

marvmena committed:
cambios finales

cambios finales

marvmena committed:
actualización

ultimos cambios

crisSotoMo committed:
cambios vinculos

marvmena committed:
Merge branch 'Desarrollo' of https://github.com/crisSotoMo/PlataformaEcommerce into Desarrollo

marvmena committed:
Cambios finales oficial

Se agregaron enlaces a footers y headers y se editaron algunas vistas.

marvmena committed:
cambios

commit

crisSotoMo committed:
agregar metodo pago comprador

marvmena committed:
Se agregó espacio a los formularios

Se agregó espacio como fue indicadó en el feedback del profesor. Se le agregó a los formularios.

marvmena committed:
API

Se agregó el codigo del API, adelanto.

Danieljm11 committed:
Actualizacion Backend

Actualizacion Backend

marvmena committed:
Modulos, personas, actualizados

Se modificaron los modulos de personas, para estandarizarle y agregar los roles

marvmena committed:
Modulos Usuario y Vendedor

Se finalizó el modulo de usuario, y se comenzó con el modulo de vendedor.

Jhnst2705 committed:
Modelo y ruta de tarjeta

Creamos el modelo y la ruta para el método de pago

marvmena committed:
Cambios de Leo. CRUDs.

Cambios de Leo. CRUDs.

marvmena committed:
Merge branch 'Desarrollo' of https://github.com/crisSotoMo/PlataformaEcommerce into Desarrollo

Jhnst2705 committed:
se crea modelo y ruta de compra, ademas se elimina idPersona de Tarjeta

Jhnst2705 committed:
Validaciones de registro de producto

marvmena committed:
Correcciones de reunión

Se agregaron algunas correcciones de las turas y modelos de JavaScript

cherloydfans13141 committed:
User menu and css fixes

All pages are now connected and a user menu has been added

marvmena committed:
Cambios Producto e Inventario de Vendedor

Se agregaron versiones iniciales de los módulos de producto y se trabajo en la unificación con las vistas

marvmena committed:
Merge branch 'Desarrollo' of https://github.com/crisSotoMo/PlataformaEcommerce into Desarrollo
